WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController.swift

```diff
@@ -410,20 +408,20 @@ class MySiteViewController: UIViewController, UIScrollViewDelegate, NoSitesViewD
         navigationController?.navigationBar.accessibilityIdentifier = "my-site-navigation-bar"
     }
 
-    private func setupNavBarAppearance() {
-        navigationController?.setNavigationBarHidden(true, animated: false)
+    private func configureNavBarAppearance(animated: Bool) {
+        if scrollView.contentOffset.y >= 60 {
+            navigationController?.setNavigationBarHidden(false, animated: animated)
```

Comment:
I'd keep a local `isNavigatinoBarHidden` variable to avoid calling it on every scroll event that gets delivered very frequently. It's just to be safe. We don't know what UIKit does internally.

Reply:
Done! https://github.com/wordpress-mobile/WordPress-iOS/pull/21585/commits/d74ffe62963a527ee6b8ed0b072ff91b080120fe

Reply:
Yes, I was testing on the WP app on iOS 17 simulator.